install/kubernetes/helm/istio/values.yaml

```diff
@@ -93,10 +93,15 @@ ingress:
 #  memory: 128Mi
   service:
     loadBalancerIP: ""
-    nodePort:
-      enabled: false
-      port: 32000
-
+    type: LoadBalancer #change to NodePort, ClusterIP or LoadBalancer if need be
+    ports:
+    - port: 80
+      name: http
+      nodePort: 32000
```

Comment:
Is it ok to add nodePort with LoadBalancer ? If yes, great. Otherwise - we need extra filtering.

Reply:
I'll do more testing thanks.

Reply:
yes, it does work.  see below:

```

~/Downloads/istio-release-0.8-20180511-16-55/install/kubernetes ⌚ 21:54:35
$ cat istio-ingress-test.yaml 
apiVersion: v1
kind: Service
metadata:
  name: istio-ingress
  namespace: istio-system
  labels:
    chart: ingress-0.8.0
    release: RELEASE-NAME
    heritage: Tiller
    istio: ingress
spec:
  # loadBalancerIP: "1.1.1.1"
  type: LoadBalancer 
  selector:
    istio: ingress
  ports:
    -
      name: http
      nodePort: 32000
      port: 80
    -
      name: https
      port: 443


$ kubectl get services -n istio-system
NAME                       TYPE           CLUSTER-IP       EXTERNAL-IP    PORT(S)                                                               AGE
istio-citadel              ClusterIP      172.21.124.13    <none>         8060/TCP,9093/TCP                                                     6h
istio-egressgateway        ClusterIP      172.21.126.139   <none>         80/TCP,443/TCP                                                        6h
istio-ingress              LoadBalancer   172.21.117.218   169.60.83.11   80:32000/TCP,443:32745/TCP                                            6s


```